### DISCUSSION TITLE
added bakabakaband to the Angband variant's table

### DISCUSSION THREAD

**Discussion by igroglaz** [sikabane-works/bakabakaband#1706] [Show and tell]
Hiho! 

I've added bakabakaband to the Angband variant's table:

https://tangaria.com/variants/

You are welcome to correct the description as we google translate stuff :) Thanks!

Kind regards, 
igroglaz

**Comment by deskull-m** [comment]
Nice to meet you.

Thank you for adding to the introduction. I'm not native English speaker, but I would like to summarize expressions about my personal variant that are less misleading than machine translation.
I will upload it here as soon as it is written.

Sincerely,
Deskull

**Reply by deskull-m** [reply]
Hello. Sorry for the long delay. I tried to write a summary in English, so if this is OK, please reflect it in your web.
https://github.com/sikabane-works/bakabakaband/blob/develop/README_en.md

Sincerely,
Deskull

**Reply by deskull-m** [reply]
P.S. If you can introduce more Japanese variants, I'd like you to add that.
http://www.miyamasa.net/heng_th_katte.html
This, "Hengband Touhou Katte Ver" is also only available in Japanese, but it is a well-made Hengband variant that has been introducing elements of Touhou Project for many years.

Regards again,
Deskull

**Comment by igroglaz** [comment]
Thanks! Added :D 

People in V community pretty interested in Japanese variants. Some topics on forum:
http://angband.oook.cz/forum/showthread.php?t=10814
http://angband.oook.cz/forum/showthread.php?t=11040

Regards,
igro
